src/couch_replicator/test/eunit/couch_replicator_retain_stats_between_job_runs.erl

```diff
@@ -254,10 +216,9 @@ replicate(Source, Target) ->
     {ok, Pid, Rep#rep.id}.
 
 scheduler_jobs() ->
-    Addr = config:get("chttpd", "bind_address", "127.0.0.1"),
-    Port = mochiweb_socket_server:get(chttpd, port),
-    Url = lists:flatten(io_lib:format("http://~s:~b/_scheduler/jobs", [Addr, Port])),
-    {ok, 200, _, Body} = test_request:get(Url, []),
+    ClusterUrl = couch_replicator_test_helper:cluster_url(),
```

Comment:
/me thinking about `couch_replicator_test_helper:cluster_url(Path)` function

```erlang
cluster_url(<<"/", _/binary>> = Path) -> 
   <<(cluster_url())/binary, Path/binary>>;
cluster_url(Path) -> 
   <<(cluster_url())/binary, "/", Path/binary>>.
```

Reply:
Good idea that work well. I'll add it.

Reply:
I think I can just re-use this suggestion for cluster_db_url as the logic the same, it just didn't flexibly handle the "/" case before.